felix/fv/iptables_cleanup_test.go

```diff
@@ -79,6 +79,55 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ iptables cleanup tests", []
 		})
 	})
 
+	Describe("switching from iptables -> nftables", func() {
```

Comment:
very much a nit: should we rename this file now that it has nftables test cases as well?